ecosystem/typescript/sdk/package.json

```diff
@@ -35,12 +35,10 @@
   "dependencies": {
     "@scure/bip39": "^1.1.0",
     "axios": "^0.27.2",
-    "buffer": "^6.0.3",
     "ed25519-hd-key": "^1.2.0",
     "js-sha3": "^0.8.0",
     "tweetnacl": "^1.0.3",
-    "typescript-memoize": "^1.1.0",
-    "yarn": "^1.22.19"
```

Comment:
This has been removed in another PR. This PR needs a rebase.

Reply:
updated @jjleng